docs/qt.rst

```diff
@@ -0,0 +1,32 @@
+# Qt
+
+For Qt-based applications we have the `org.kde.Sdk` that will offer us most Qt
+modules and KDE Frameworks for our applications to use.
+
+For example, the following JSON makes building a random Qt application really
+straight-forward.
+
+.. code-block:: json
+
+    {
+      "id": "org.kde.blinken",
+      "branch": "master",
+      "runtime": "org.kde.Platform",
+      "runtime-version": "5.9",
+      "sdk": "org.kde.Sdk",
+      "command": "blinken",
+      "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland" ],
+
+      "modules": [
+        {
+          "name": "blinken",
+          "buildsystem": "cmake-ninja",
```

Comment:
`"builddir": true`

Reply:
` "-DCMAKE_BUILD_TYPE=Release"` or `"-DCMAKE_BUILD_TYPE=RelWithDebInfo"` ?

Reply:
`"-DCMAKE_INSTALL_LIBDIR=lib"` (workaround for flatpak+ECM bug)

Reply:
What bug?

Reply:
This is supposed to be fixed: https://github.com/flatpak/flatpak-builder/issues/93
But projects using extra-cmake-modules still install in /lib64.

Reply:
The ECM bug is a bug, that only affects projects using ECM. Instead of documenting it I'd rather have it fixed.

Reply:
https://github.com/flatpak/freedesktop-sdk-base/issues/22